state_machines/aws_lambda_power_tuning_state_machine.json

```diff
@@ -5,29 +5,47 @@
         "Initializer": {
             "Type": "Task",
             "Resource": "${initializerArn}",
-            "Next": "Branching",
```

Comment:
Please remember that the state machine file is now available in the main repo, so we don't need to duplicate it here :)